draft-irtf-cfrg-hpke.md

```diff
@@ -1207,6 +1207,27 @@ across all labels in this document.
 The `0xFFFF` AEAD ID is reserved for applications which only use the Export
 interface; see {{hpke-export}} for more details.
 
+# API Considerations {#api-considerations}
+
+The high-level HPKE APIs specified in this document are all fallible.
+For example, `Decap()` can fail if the encapsulated key `enc` is invalid,
+and `Open()` may fail if ciphertext decryption fails. This specification
+notes cases where explicit errors may occur.
+
+However, implicit errors may also occur. As an example, certain classes of
+failures, e.g., malformed recipient public keys, may not yield explicit
+errors. For example, for the DHKEM variant described in this specification,
+the `Encap()` algorithm may fail when given an invalid recipient public key.
+However, other KEM algorithms may not have an efficient algorithm for verifying
+the validity of public keys. As a result, an equivalent error may not manifest
+until AEAD decryption at the recipient. As another example, the `AuthDecap()`
```

Comment:
```suggestion
until AEAD decryption at the recipient. As another example, DHKEM's `AuthDecap()`
```
(clarify that we are still talking about DHKEM, as some future KEMs might be able to detect wrong sender public keys explicitly)